src/Games/brain-calc.js

```diff
@@ -0,0 +1,44 @@
+import getRandomNumber from '../randomCount.js';
+import gamesCore from '../index.js';
+
+const descriptionOfGame = 'What is the result of the expression?';
+
+const getRandomOperation = () => {
+  const operators = ['+', '-', '*'];
+  const choiceRandomOperatorIndex = getRandomNumber(0, 2);
+  const choisedRandomOperator = operators[choiceRandomOperatorIndex];
+  return choisedRandomOperator;
```

Comment:
return operators[choiceRandomOperatorIndex];

Reply:
at first it was like You write, but I changed it for more beauty ))
fixed